STM32F405RGTx/Core/Src/rtos_threads/measurements_thread.cpp

```diff
@@ -5,12 +5,26 @@
 #include "mc.hpp"
 #include "cmsis_os.h"
 #include "threads.hpp"
+#include "lut.hpp"
 
-// TODO: add defines here for conversions..
+#define CURRENT_SENSE_RESISTANCE                1E-3
+#define ADC_TO_VOLTAGE(adc)                     ( adc * (3.3/(1 << 12)) )
+#define VOLTAGE_TO_CURRENT(voltage)             (voltage / CURRENT_SENSE_RESISTANCE)
```

Comment:
Whenever you define a macro like this, the argument for the macro should be enclosed by brackets, since all that happens with a macro is glorified copy and paste:

```C
#define ADC_TO_VOLTAGE(adc)    ( (adc)*(3.3/(1 << 12)) )
```